lib/DB/Schema/ResultSet/ProblemSet.pm

```diff
@@ -9,6 +9,7 @@ use base 'DBIx::Class::ResultSet';
 
 use Carp;
 use Clone qw/clone/;
+use Mojo::JSON qw/true false/;
```

Comment:
I don't think that these imports are used anywhere in this file.  So this should be removed.